build_projects/shared-build-targets-utils/Utils/Monikers.cs

```diff
@@ -33,12 +33,37 @@ public static string GetHostFxrBrandName(BuildTargetContext c)
             return GetBrandName(c, "Host FX Resolver");
         }
 
+        private static bool IsPortableRID(string rid, out string portablePlatformID)
```

Comment:
This is duplicated with the method in SharedFrameworkPublisher.  Can it be refactored to remove duplication?